__tests__/app.test.js

```diff
@@ -78,4 +110,15 @@ describe("app", () => {
       });
     });
   });
+  describe("ALL /notapath", () => {
```

Comment:
Nicely done, well done on putting this in it's own describe block, as it's not specific to just one ticket 👍